### DISCUSSION TITLE
Counting objects

### DISCUSSION THREAD

**Discussion by Ysbrand** [blakeblackshear/frigate#2558] [Ask A Question]
I've tried to find an answer, but failed....

I'm trying to count the number of cars detected by Frigate on a daily basis (HA). 

Initially I tried the binary sesnsor, but that will give me only a partial result as more than one car can pass by while the binary sensor is "on"

So I guess I need to use the regular sensor.area_car, but how?

Thanks

**Comment by NickM-27** [comment]
Yeah, it would be difficult too since I'm not sure when you would aggregate it. For example if it said it currently sees 3 cars and then changes to 2. Does that mean it has seen 3 unique cars or 5? My solution would be a combination of the binary sensor and the general sensor.

**Reply by Ysbrand** [reply]
This is wat I've currently got:

```
- platform: history_stats
  name: straat_car_count_daily
  entity_id: binary_sensor.straat_car_motion
  state: "on"
  type: count
  start: "{{ now().replace(hour=0, minute=0, second=0) }}"
  end: "{{ now() }}"

- platform: history_stats
  name: straat_car_count_hourly
  entity_id: binary_sensor.straat_car_motion
  state: "on"
  type: count
  start: "{{ now().replace(minute=0, second=0) }}"
  end: "{{ now() }}"
```

But I'm not sure if the binary sensor counts every car, in other words, when one car is detected and the second one appears before the first one disappears; will that flip the binary sensor twice or only once?
